packages/web/src/features/translation/TranslationSidebar.tsx

```diff
@@ -68,20 +64,20 @@ export const TranslationSidebar = ({
       </div>
       <div className="grow flex flex-col min-h-0">
         <Tab.Group>
-          <Tab.List className="flex flex-row gap-1 border-b-2 border-slate-800 md:-ms-3 -mx-4 px-3">
+          <Tab.List className="flex flex-row md:-ms-3 -mx-4 px-3">
```

Comment:
What do you think about removing the px-3 here so that the tab bottom line goes to the edge?